DESCRIPTION

```diff
@@ -28,17 +28,16 @@ Imports:
     hardhat (>= 1.1.0),
     magrittr,
     purrr,
-    rlang,
+    rlang (>= 1.1.0),
     rpart,
     rsample,
     tibble,
     tidyr,
     utils,
     withr
 Suggests: 
-    AmesHousing,
```

Comment:
i'm going to assume that this is leftover from a long time ago, and that we don't need it since we have modeldata